app/connectors/PayApiConnector.scala

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package connectors
+
+import config.ViewConfig
+import javax.inject.{Inject, Singleton}
+import model.Vrn
+import model.payapi.{SpjRequestBtaVat, SpjResponse}
+import play.api.{Configuration, Logger}
+import play.api.mvc.Request
+import uk.gov.hmrc.play.bootstrap.config.ServicesConfig
+import uk.gov.hmrc.play.bootstrap.http.HttpClient
+
+import scala.concurrent.{ExecutionContext, Future}
+
+@Singleton
+class PayApiConnector @Inject() (
+    servicesConfig: ServicesConfig,
+    httpClient:     HttpClient,
+    configuration:  Configuration,
+    viewConfig:     ViewConfig)
+  (implicit ec: ExecutionContext) {
+
+  import req.RequestSupport._
+
+  private val serviceUrl: String = servicesConfig.baseUrl("pay-api")
+  private val viewUrl: String = configuration.get[String]("microservice.services.pay-api.sj-url")
+
+  def startJourney(amountInPence: Long, vrn: Vrn)(implicit request: Request[_]): Future[SpjResponse] = {
+
+    val bUrl: String = s"${viewConfig.frontendBaseUrl}${configuration.get[String]("urls.pay-back-url")}${vrn.value}"
```

Comment:
Here, the  config is read during runtime.
`configuration.get[String]("urls.pay-back-url")`
If the values are missing in the config file, it will throw exception. Could you extract it to the class level, so it will be read during startup? 
Thanks to that, if it will fail, it will fail sooner.

Reply:
ok